tools/SourceKit/lib/SwiftLang/SwiftLangSupport.cpp

```diff
@@ -623,10 +623,47 @@ Optional<UIdent> SwiftLangSupport::getUIDForDeclAttribute(const swift::DeclAttri
         return Attr_Objc;
       }
     }
+    case DAK_AccessControl: {
+      static UIdent Attr_Private("source.decl.attribute.private");
+      static UIdent Attr_FilePrivate("source.decl.attribute.fileprivate");
+      static UIdent Attr_Internal("source.decl.attribute.internal");
+      static UIdent Attr_Public("source.decl.attribute.public");
+      static UIdent Attr_Open("source.decl.attribute.open");
```

Comment:
Any reason we are not adding these to ProtocolUIDs.def?

Reply:
Attributes are defined in `Attr.def`. The way that AccessLevel attributes work is via `DECL_ATTR_ALIAS`, which forces us to explicitly check the AccessLevel and pick the right UID.

Reply:
ah, OK. Thanks for the explanation!